float64 depth_desired
uint16 pwm_max 
uint16 pwm_neutral
uint32 K
uint32 KI
uint32 KP
uint32 KD
uint16 rosrate
